### PR TITLE
Add recipe for aiapy

### PR DESCRIPTION
**Pull request by wtbarnes**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by wtbarnes** [comment]
This is blocked by #13355

File: recipes/aiapy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - importlib_resources  # [py<37]
```
Comment (dopplershift) [inline recipes/aiapy/meta.yaml:24]: It's safe to just have this dependency on `importlib_resources` without the selector. Yes, it's redundant with the standard library package on Python >= 3.8, but then you can make this `noarch:python`, which is a big win.

File: recipes/aiapy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (dopplershift) [inline recipes/aiapy/meta.yaml:21]: When making noarch, make this `python >=3.6`.

File: recipes/aiapy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - setuptools_scm
+  run:
+    - importlib_resources  # [py<37]
+    - python
```
Comment (dopplershift) [inline recipes/aiapy/meta.yaml:25]: `python >=3.6` here too

File: recipes/aiapy/meta.yaml (hunk 2 above)
Comment (wtbarnes) [inline recipes/aiapy/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/aiapy/meta.yaml (hunk 1 above)
Comment (wtbarnes) [inline recipes/aiapy/meta.yaml:24]:
```suggestion
    - importlib_resources
```

File: recipes/aiapy/meta.yaml (hunk 3 above)
Comment (wtbarnes) [inline recipes/aiapy/meta.yaml:25]:
```suggestion
    - python >=3.6
```

File: recipes/aiapy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiapy-{{ version }}.tar.gz
+  sha256: 6937fc6ba3e931f2069baea4d5d2153122a5a6f0f6de9721f31733f12eecd18e
+
+build:
+  number: 0
+  skip: true   # [py2k]
```
Comment (wtbarnes) [inline recipes/aiapy/meta.yaml:15]:
```suggestion
  skip: true   # [py2k]
  noarch: python
```

File: recipes/aiapy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
```
Comment (wtbarnes) [inline recipes/aiapy/meta.yaml:15]:
```suggestion
```

**Comment by wtbarnes** [comment]
Thanks for the feedback @dopplershift! I think I've addressed all of your comments. I had to remove the `skip: true` and add a `noarch: python` to make the linter happy as well.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**